src/main/java/org/whispersystems/textsecuregcm/storage/Accounts.java

```diff
@@ -76,6 +76,9 @@
   @SqlQuery("SELECT * FROM accounts")
   public abstract Iterator<Account> getAll();
 
+  @SqlQuery("SELECT " + NUMBER + " FROM accounts WHERE " + ID + " & " + ":mask = :bucket")
```

Comment:
In our testing this query performed badly? I thought we were going to do the key reference thing

Reply:
changed in 4d5b40bb442c40bcda99aaa1748b6581875479b8